src/test/scala/uk/gov/hmrc/test/AkkaMaterializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
What I am trying to understand is why `AkkaMaterializerSpec` has been included? Can you not just call the html fragments directly and override the`assetsConfig` and `optimizelyConfig` implicits?
```
@head(linkElem = None, headScripts = Some(headScripts))(assetsConfig, optimizelyConfig)
```
```
@footer(
 analyticsToken = None,
 analyticsHost = "",
 ssoUrl = None,
 scriptElem = Some(footerScriptElem),
 gaCalls = None)(assetsConfig, optimizelyConfig)

```
 In the same fashion as [InputRadioGroupSpec](https://github.com/hmrc/play-ui/blob/master/src/test/scala/uk/gov/hmrc/play/views/helpers/InputRadioGroupSpec.scala#L45)

Then if I have understood correctly there would be no need for the `TestWithoutPlayAppController.scala`, `AkkaMaterializerSpec.scala` or the `test_header_footer_includes.scala.html`